### ISSUE TITLE
Typings for `Description` component are incorrect

### ISSUE THREAD

**Issue by itshaydendev** [zeit-ui/react#85]
# Bug report 🐞

When I try and embed a component in the `content` attribute, like this:

```jsx
<Description title={field[0]} content={<Text p>{field[1]}</Text>} />
```

It gives me this error:

```
Type 'Element' is not assignable to type 'string'.
```

## Version & Environment

```json
"@zeit-ui/react": "^0.0.1-beta.22",
...
"react": "^16.13.1",
"react-dom": "^16.13.1",
```

**Comment by unix** [comment]
Thanks for your feedback, this is because the union type is overwritten, I fixed the bug here: #86 .

**Comment by unix** [comment]
The CI will release a new beta version later, please try to upgrade the dependencies.
Any updates, please let me know.
